_data/simple-icons.json

```
@@ -3184,6 +3184,11 @@
             "hex": "47848F",
             "source": "https://www.electronjs.org/"
         },
+        {
+            "title": "Electron Builder",
```

It seems the official spelling is "electron-builder".

```suggestion
            "title": "electron-builder",
```